yuuki/shell.py

```diff
@@ -43,6 +43,7 @@ def main():
             break
         except KeyboardInterrupt:
             print "^C"
+            break
```

Comment:
Most interactive interpreters/shells (e.g. bash, python, irb) don't exit on Ctrl-C; they just cancel the current command. I'd prefer to keep Ctrl-D as exit.